### PR TITLE
feat: support for new billable metrics

### PR DESCRIPTION
**Pull request by kevcodez**
This can already be merged without API changes, as the metrics won't be rendered unless output by the API

### CODE REVIEW

File: apps/studio/components/interfaces/Organization/BillingSettings/BillingBreakdown/BillingBreakdown.constants.ts [hunk 1]
```diff
@@ -82,11 +90,46 @@ export const BILLING_BREAKDOWN_METRICS: Metric[] = [
...
+  {
+    key: PricingMetric.DISK_THROUGHPUT_GP3,
+    name: 'Disk Throughput',
+    units: 'absolute',
+    unitName: 'IOPS-Hrs',
+    category: 'Database',
+  },
+  {
+    key: PricingMetric.LOG_DRAIN_MESSAGES,
+    name: 'Log Drain Messages',
```
Comment (Ziinc) [inline apps/studio/components/interfaces/Organization/BillingSettings/BillingBreakdown/BillingBreakdown.constants.ts:130]:
```suggestion
    key: PricingMetric.LOG_DRAIN_EVENTS,
    name: 'Log Drain Events',
```

File: apps/studio/data/analytics/org-daily-stats-query.ts [hunk 2]
```diff
@@ -33,6 +35,11 @@ export enum PricingMetric {
   PITR_7 = 'PITR_7',
   PITR_14 = 'PITR_14',
   PITR_28 = 'PITR_28',
+  DISK_IOPS_GP3 = 'DISK_IOPS_GP3',
+  DISK_IOPS_IO2 = 'DISK_IOPS_IO2',
+  DISK_THROUGHPUT_GP3 = 'DISK_THROUGHPUT_GP3',
+  LOG_DRAIN = 'LOG_DRAIN',
+  LOG_DRAIN_MESSAGES = 'LOG_DRAIN_MESSAGES',
```
Comment (Ziinc) [inline apps/studio/data/analytics/org-daily-stats-query.ts:42]:
```suggestion
  LOG_DRAIN_EVENTs = 'LOG_DRAIN_EVENTS,
```

**Review by alaister — APPROVED** [review]
LGTM!

- Opened staging preview and viewed billing usage pages

File: apps/studio/components/interfaces/Organization/BillingSettings/BillingBreakdown/BillingBreakdown.constants.ts [hunk 3]
```diff
@@ -45,6 +45,14 @@ export const BILLING_BREAKDOWN_METRICS: Metric[] = [
     category: 'Authentication',
     unitName: 'MAU',
   },
+  {
+    key: PricingMetric.MONTHLY_ACTIVE_THIRD_PARTY_USERS,
+    name: 'Monthly Active Third-Party Users',
+    units: 'absolute',
+    anchor: 'mauSso',
```
Comment (ecktoteckto) [inline apps/studio/components/interfaces/Organization/BillingSettings/BillingBreakdown/BillingBreakdown.constants.ts:52]: **question:** Haven't checked the concept of "anchors", but is "...Sso" correct?

File: apps/studio/components/interfaces/Organization/BillingSettings/BillingBreakdown/BillingBreakdown.constants.ts [hunk 4]
```diff
@@ -82,11 +90,46 @@ export const BILLING_BREAKDOWN_METRICS: Metric[] = [
     category: 'Edge Functions',
   },
   {
-    key: PricingMetric.DISK_SIZE_GB_HOURS,
+    key: PricingMetric.DISK_SIZE_GB_HOURS_GP3,
     name: 'Disk Size',
```
Comment (ecktoteckto) [inline apps/studio/components/interfaces/Organization/BillingSettings/BillingBreakdown/BillingBreakdown.constants.ts:94]: **question:** We don't want to have the "GP3" suffix here?

**Pull request merged**